pyproject.toml

```diff
@@ -7,6 +7,8 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = ">=3.7,<4.0"
+libcst = "^0.4.0"
+typer = "^0.4.0"
```

Comment:
You may want to include the "all" extra (see the Typer [documentation](https://typer.tiangolo.com/tutorial/package/#dependencies-and-environment)). This enables some extra `Click` support, and a few other smaller features.